.github/workflows/cron.yml

```diff
@@ -0,0 +1,44 @@
+name: Daily cron test
```

Comment:
do we need to copy/paste? There's no way to do inheritance / includes?

Reply:
Not that I know of. If I'm wrong, that would be great.